fix: don't show double vertical scrollbars on sbtx file

geminicaprograms opened the pull request:
According to [SO](https://stackoverflow.com/a/61844861) it was determined that the top container has to have
fixed height (could be `100vh` related but cannot be `100%` related) and
then inner container can have 100% height. Note that `minHeight` was set to `450px`.

Apart from the the following changes were applied
* all margins were reduced so that file/code areas use more screen
  estate
* selected file (in file view) is no longer bolded (but still
  highlighted so that there is no doubt that it is selected) so that it
  doesn't take more space
* proportion between file and content views was modified so that more
  space is available for file view (potentially reducing the need for
  horizontal scrollbar)

Here is the current look & feel for `1794px` width:
![image](https://user-images.githubusercontent.com/3641082/194550388-a6c26f26-b139-4b57-a9e4-6169890bfd88.png)

Closes #92

adamw commented:
Great :) though I guess the no-scrollbar thing depends on the screen width?

geminicaprograms commented:
> Great :) though I guess the no-scrollbar thing depends on the screen width?

yeap, it will show-up anyway if the screen is small enough and we can't do much about it :/ the good thing is that by _default_ (in your and mine screens ;)) it will not show 😸

adamw commented:
Turns out my screen is 1792 wide ;-) 

Anyway, reading through my original issue, the problem is with the *vertical* scroll. Try opening `sbtx`, for example, and you get two vertical scrolls: one for the code viewer, and one for the entire window - the code viewer gets stretched. The second one should be there.

As for the changes here: the code is unfortunately too small now, it looks weird. So I would keep the current size. Let's keep the smaller margins - no reason not to use as much screen as we can.

geminicaprograms commented:
> Turns out my screen is 1792 wide ;-)
> 
> Anyway, reading through my original issue, the problem is with the _vertical_ scroll. Try opening `sbtx`, for example, and you get two vertical scrolls: one for the code viewer, and one for the entire window - the code viewer gets stretched. The second one should be there.
> 
> As for the changes here: the code is unfortunately too small now, it looks weird. So I would keep the current size. Let's keep the smaller margins - no reason not to use as much screen as we can.

I have reworked it so that `sbtx` file (and others too) are displayed correct. As suggested:
* kept the bigger screen estate to be utilised
* removed content view font size reduction

adamw commented:
Works great now, thanks :)
